playground/vitest.config.ts

```diff
@@ -12,7 +12,8 @@ export default defineVitestConfig({
         rootDir: fileURLToPath(new URL('./', import.meta.url)),
         domEnvironment: process.env.VITEST_DOM_ENV as 'happy-dom' | 'jsdom' ?? 'happy-dom',
       },
-    }, 
-    setupFiles: ['./tests/setup/mocks.ts']
+    },
+    setupFiles: ['./tests/setup/mocks.ts'],
+    globals: true
```

Comment:
Testing-library relies on testing globals being available to do automatic cleanup. I had to change this accordingly.